SU-SWS/stanford_paragraph_types
# EARTH-000: No tint option defaults when set to 0

# READY FOR REVIEW

# Summary
- Allows the no-tint option to work.
- Also, changed the schema version of the update hooks so that it doesn't think we are on 2.x.
- Will need to manually change the schema on the production database.

# Needed By (Date)
- EO Sprint

# Urgency
- Not

# Steps to Test

1. Check out this branch
2. Run drush updb
3. Revert the feature
4. Add a no-tint paragraph :D

# Associated Issues and/or People
- EARTH-404

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
